accounts/signals.py

```diff
@@ -1257,8 +1257,8 @@ def post_migrate_create_flatpages(sender, **kwargs):
         			<h3>Questions about the studies</h3>
         			<p>If you have any questions or concerns about the studies, please first check our FAQ. With any additional questions, please contact us at <a href="mailto:lookit@mit.edu">lookit@mit.edu</a>.</p>
         			<h3>For researchers</h3>
-        			<p>We are committed to making Lookit a tool for all developmental researchers and to keeping the code open-source. You can view the current <a href="https://github.com/CenterForOpenScience/lookit" target="_blank">Lookit</a>, <a href="https://github.com/CenterForOpenScience/experimenter" target="_blank">Experimenter</a>, and <a href="https://github.com/CenterForOpenScience/exp-addons" target="_blank">exp-addons</a> (specific frames for Lookit studies) codebases on github; <a href="https://github.com/kimberscott?tab=repositories" target="_blank">Kim's forks</a> may have more up-to-date versions of data analysis code.</p>
-        			<p>We're not quite ready to start hosting additional studies from other labs yet, but our partners at the Center for Open Science are actively developing functionality for multiple labs to create and post their own studies on Lookit. Although we aim for the platform to be as accessible as possible, some familiarity with programming basics will be helpful for researchers. An ideal skillset for creating your own studies would include basic understanding of HTML, CSS, and JavaScript for study creation; passing familiarity with the <a href="https://guides.emberjs.com/v2.8.0/" target="_blank">Ember framework</a> used by Lookit and Experimenter; and python for data processing.</p>
+        			<p>We are committed to making Lookit a tool for all developmental experimenters and to keeping the code open-source. You can view the current <a href="https://github.com/CenterForOpenScience/lookit" target="_blank">Lookit</a>, <a href="https://github.com/CenterForOpenScience/experimenter" target="_blank">Experimenter</a>, and <a href="https://github.com/CenterForOpenScience/exp-addons" target="_blank">exp-addons</a> (specific frames for Lookit studies) codebases on github; <a href="https://github.com/kimberscott?tab=repositories" target="_blank">Kim's forks</a> may have more up-to-date versions of data analysis code.</p>
+        			<p>We're not quite ready to start hosting additional studies from other labs yet, but our partners at the Center for Open Science are actively developing functionality for multiple labs to create and post their own studies on Lookit. Although we aim for the platform to be as accessible as possible, some familiarity with programming basics will be helpful for experimenter. An ideal skillset for creating your own studies would include basic understanding of HTML, CSS, and JavaScript for study creation; passing familiarity with the <a href="https://guides.emberjs.com/v2.8.0/" target="_blank">Ember framework</a> used by Lookit and Experimenter; and python for data processing.</p>
```

Comment:
just for clarity can this stay as it was?

Reply:
Actually, I think I understand what you mean.  I'll put up a commit and take my chances.